api/src/main/resources/workbench-api.yaml

```diff
@@ -804,16 +804,22 @@ paths:
           description: Workspace deletion request accepted
           schema:
             "$ref": "#/definitions/EmptyResponse"
-  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cdr-selectors":
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/workspaceResources":
```

Comment:
nit (if there's time): I'd probably go with just `resources` as the path segment, since we have it under `workspaces` anyway.